notebooks/tel_and_site/subsys_req_ver/m2/m2_functional_verification/LVV-T1782_M2_RMP_functionality_check_small_range.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.            await asyncio.sleep(5.0)
</pre><p>You can check the value in a faster pace such as every 1.0 second.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
But your above logic has the maximum time of 5.0 already...